tools/fp20compiler/ts1.0_inst_list.cpp

```diff
@@ -42,10 +42,18 @@ void InstList::Invoke()
 {
     int i;
 
-    printf("pb_push1(p, NV097_SET_SHADER_OTHER_STAGE_INPUT,\n"
-           "    MASK(NV097_SET_SHADER_OTHER_STAGE_INPUT_STAGE1, 0)\n"
-           "    | MASK(NV097_SET_SHADER_OTHER_STAGE_INPUT_STAGE2, 1)\n"
-           "    | MASK(NV097_SET_SHADER_OTHER_STAGE_INPUT_STAGE3, 2));\n");
+    assert(size > 0);

```

Comment:
nit: This should be `size > 1`, otherwise the loop below won't do a single iteration, so the second argument to `pb_push1` would be empty (syntax-error).